src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java

```diff
@@ -39,18 +40,13 @@
 import com.google.gwt.event.shared.HandlerRegistration;
 import com.google.gwt.user.client.Command;
 import com.google.gwt.user.client.Timer;
-import com.google.gwt.user.client.ui.RootPanel;
-import com.google.gwt.user.client.ui.ScrollPanel;
-import com.google.gwt.user.client.ui.Widget;
+import com.google.gwt.user.client.Window;
+import com.google.gwt.user.client.ui.*;
```

Comment:
Did your IDE do this? We generally avoid wildcard imports (See https://github.com/rstudio/rstudio-pro/wiki/Coding-Standards)

Reply:
Yeah, IntelliJ changes it to wildcard if you import everything in the package. It's smart about not importing too much/too little. I can change the setting though.

Reply:
In IntelliJ prefs: Editor/Code Style/Java/Imports, and increase number for "Class count to use import with '*':" to something like 99 to prevent this altogether.